optional/unbound/unbound.conf

```diff
@@ -1,19 +1,21 @@
 server:
   verbosity: 1
   interface: 0.0.0.0
-  interface: ::0
+  {{ 'interface: ::0' if SUBNET6 }}
   logfile: ""
   do-ip4: yes
-  do-ip6: yes
+  do-ip6: {{ 'yes' if SUBNET6 else 'no' }}
   do-udp: yes
   do-tcp: yes
   do-daemonize: no
   access-control: {{ SUBNET }} allow
+  {{ 'access-control: {{ SUBNET6 }} allow' if SUBNET6 }}
   directory: "/etc/unbound"
   username: unbound
   auto-trust-anchor-file: trusted-key.key
   root-hints: "/etc/unbound/root.hints"
   hide-identity: yes
   hide-version: yes
-  max-udp-size: 4096
-  msg-buffer-size: 65552
+  cache-min-ttl: 300
+  qname-minimisation: yes
```

Comment:
this is not necessary since qname-minimisation is enabled by default - quoting from the unbound.conf manpage:

```
qname-minimisation: <yes or no>
       Send minimum amount of information to upstream servers to
       enhance privacy.  Only send minimum required labels of the QNAME
       and set QTYPE to A when possible. Best effort approach; full
       QNAME and original QTYPE will be sent when upstream replies with
       a RCODE other than NOERROR, except when receiving NXDOMAIN from
       a DNSSEC signed zone. Default is yes.
```

Reply:
Good catch, my config hasn't been updated since qname-minimisation-strict has been introduced :$